articles/virtual-machines/workloads/sap/deployment-guide.md

```diff
@@ -997,6 +995,9 @@ NOTE: To install AEM Extension 0.2.2 or higher, the Latest version of AZ CLI mus
 
    1. Install Azure CLI AEM Extension
   
++ >[!Note]:Install the lastest Azure CLI 2.0, as described in [Install Azure CLI 2.0][azure-cli-2]. Make sure to use at least version 2.19.1 or higher +
```

Comment:
Please don't use NOTE section. IMHO this reduces the readability. Changing the text as described above should be good enough.